gio/ginetaddress.c

```diff
@@ -521,9 +540,15 @@ g_inet_address_new_from_string (const gchar *string)
   g_networking_init ();
 
   if (inet_pton (AF_INET, string, &in_addr) > 0)
+  {
     return g_inet_address_new_from_bytes ((guint8 *)&in_addr, AF_INET);
+  }
+#ifdef HAVE_IPV6
   else if (inet_pton (AF_INET6, string, &in6_addr) > 0)
+  {
     return g_inet_address_new_from_bytes ((guint8 *)&in6_addr, AF_INET6);
+  }
```

Comment:
Let's not add this block, want to keep changes on top of upstream code to a minimum.

Reply:
See previous.